src/main/java/com/cloudbees/jenkins/support/filter/ContentFilters.java

```diff
@@ -40,6 +42,7 @@
  */
 @Extension
 @Restricted(NoExternalUse.class)
+@Symbol("anonymizeSupportBundle")
```

Comment:
I think this might mean we need to add a dependency against structs.

Reply:
Added